### PR TITLE
Add market info on projects

### PR DESCRIPTION
**Pull request by carloslopes**

### CODE REVIEW

**Comment by carloslopes** [comment]
The issues pointed by CodeClimate are false positives

File: db/migrate/20220905200117_add_market_to_projects.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddMarketToProjects < ActiveRecord::Migration[7.0]
+  def change
+    add_column :projects, :market, :integer
```
Comment (tonyaraujop) [inline db/migrate/20220905200117_add_market_to_projects.rb:3]: Why not to also add for this migration `null: false` in order to have validation in database level?

File: app/models/project.rb [hunk 2]
```diff
@@ -1,17 +1,21 @@
 # frozen_string_literal: true
 
 class Project < ApplicationRecord
+  enum :market, %i[internal international]
```
Comment (adbatista) [inline app/models/project.rb:4]: we have enumerize in the project that already supports i18n, please use it to keep consistent with other models of the project

**Review by adbatista — DISMISSED** [review]

**Comment by adbatista** [comment]
can you solve the conflicts please?

**Pull request merged**